tasmota/tasmota_template.h

```diff
@@ -1236,7 +1242,9 @@ const uint16_t kGpioConvert[] PROGMEM = {
   AGPIO(GPIO_IEM3000_TX),     // IEM3000 Serial interface
   AGPIO(GPIO_IEM3000_RX),     // IEM3000 Serial interface
   AGPIO(GPIO_ZIGBEE_RST),     // Zigbee reset
-  AGPIO(GPIO_DYP_RX)
+  AGPIO(GPIO_DYP_RX),
+  AGPIO(GPIO_AS608_TX),
+  AGPIO(GPIO_AS608_RX)
```

Comment:
Do NOT add this. Not needed and by this message I hoped anyone would not make any changes below that line:
```
/*********************************************************************************************\
 * ATTENTION: No user changeable features beyond this point - do not add templates !!!
\*********************************************************************************************/
```